utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -242,14 +256,28 @@ run $CLUSH_EXE $CLUSH_OPTS -l root -w $SERVERS_LIST rm -f /etc/ld.so.cache
 run $CLUSH_EXE $CLUSH_OPTS -l root -w $SERVERS_LIST ldconfig
 {% endif %}
 
-if [[ $MPICH_PATH ]] ; then
+if [[ ! -f "$DAOS_INSTALL_DIR/bin/activate" ]] ; then
+	info "Installing avocado"
+	run $PYTHON_EXE -m venv "$DAOS_INSTALL_DIR"
+	source "$DAOS_INSTALL_DIR/bin/activate"
+	run $PIP_EXE install --upgrade pip
+	run $PIP_EXE install -r "$DAOS_SOURCE_DIR/requirements.txt"
```

Comment:
After my change this could install fewer packages.
```suggestion
	run $PIP_EXE install -r "$DAOS_SOURCE_DIR/requirements-build.txt"
```

Reply:
I started with requirements-ftest.txt, and I thought that what can do more, can do less.  This is why I switched to the full install.  I am also hopping that other tasks than running ftest could be perform with installing all the python packages.
However, I have no strong opinion on this point and thus I will do what you think is better.

Reply:
The disadvantage of using `requirements.txt` is that more software will be installed and therefore it'll take slightly longer and use minimally more bandwidth and disk space.  If you're running the build, NLT (which counts as unit testing) and ftest from the same virtual environment then this is almost correct - you'll probably have a package or two that are only used for interacting with GitHub but the alternative would be to use 3 different requirements files here which would be a mess as well.

I only mentioned it as the behaviour/requirements of this file has changed since you started this PR and I wanted to let you know that you might need to re-evaluate this.  It sounds like no change is needed.

Reply:
From my side, I would prefer to keep as it is: the overhead (i.e. disk usage and overhead time) compared to the build process of DAOS and dependencies is neglectable.
I confirm that one aim of this ansible stuff is to be able to build DAOS, run ftest and run NLT with the same virtual env.